tests/unit/common/uuidGeneratorHelper.test.js

```diff
@@ -0,0 +1,17 @@
+import { getDeviceId } from "../../../src/js/common/uuidGeneratorHelper";
+import * as uuidGeneratorHelper from "../../../src/js/common/uuidGeneratorHelper";
+
+describe("uuidGeneratorHelper", () => {
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    it("getDeviceId", () => {
+        jest.spyOn(uuidGeneratorHelper, "getDeviceId").mockReturnValue("424f48aa-b723-4f97-8a30-d214b43bf372");
```

Comment:
This test doesn't really do anything. You are asserting that your spy returns the value you tell it to return. You should stub/mock/spy `uuid` package itself

Reply:
done